### PR TITLE
Sebastian/refine repo and readme

### PR DESCRIPTION
**Pull request by LaverdeS**
added some badges and emojis to readme as part of the repo refinement but:
- Maybe there is overuse of badges here but I put the one I thought were the moro relevant.
- Python 3.8 and 3.9 comes comes the pypip package but I could leave just 3.8

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,13 +5,24 @@
   >
 </h3>
 
+<div align="center">
+
+  <a href="">![https://pypi.python.org/pypi/unstructured/](https://img.shields.io/pypi/l/unstructured.svg)</a>
+  <a href="">![https://pypi.python.org/pypi/unstructured/](https://img.shields.io/pypi/pyversions/unstructured.svg)</a>
+  <a href="">![https://GitHub.com/unstructured-io/unstructured.js/graphs/contributors](https://img.shields.io/github/contributors/unstructured-io/unstructured)</a>
+  <a href="">![code_of_conduct.md](https://img.shields.io/badge/Contributor%20Covenant-2.1-4baaaa.svg) </a>
```
Comment (cragwolfe) [inline README.md:13]: code_of_conduct.md does not exist in this repo yet so broken link

Comment (cragwolfe) [reply README.md:13]: ah, nvm, see all links are not filled in yet.

Comment (LaverdeS) [reply README.md:13]: thanks, just recently added it

Comment (LaverdeS) [reply README.md:13]: [CODE_OF_CONDUCT.md](https://github.com/Unstructured-IO/unstructured/blob/main/CODE_OF_CONDUCT.md)

Comment (LaverdeS) [reply README.md:13]: oh I will fix the links then thanks

Comment (LaverdeS) [reply README.md]: DONE

Comment (LaverdeS) [reply README.md]: shall we count this as done? @MthwRobinson @cragwolfe

Comment (MthwRobinson) [reply README.md]: Looks good to me!

**Review by MthwRobinson — APPROVED** [review]
LGTM! Love the README refresh

**Pull request merged**